lawnchair/res/values/strings.xml

```diff
@@ -60,6 +60,8 @@
     <string name="transparent_background_icons">Transparent Themed Icons</string>
     <string name="transparent_background_icons_description">Use transparent background on themed icons</string>
     <string name="smartspace_mode_selection">At a Glance Provider selection</string>
+    <string name="pref_separate_recents_activity_title">Separate Recents Activity</string>
+    <string name="pref_separate_recents_activity_description">Uses a separate recents activity instead of using the integrated recents+Launcher UI (GesturNavContract API)</string>
```

Comment:
Users will have no idea what GestureNavContract means, so I've also clarified the description.
```suggestion
    <string name="pref_separate_recents_activity_description">Improves app animations by interfacing with the system launcher. Not all OEMs support it.</string>
```